### DISCUSSION TITLE
How to run the game on Mac

### DISCUSSION THREAD

**Discussion by NirbhaySharma504** [LN-Studios/Matchup-League#6] [Q&A]
Hi there
I was trying to run the project on my machine locally using maven, but when I do "mvn spring-boot:run" it gives error:

[ERROR] No plugin found for prefix 'spring-boot' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/Users/nirbhay/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]

Thanks

**Comment by freakun0025** [comment]
yeah i also have the same doubt

**Comment by lognog2** [comment]
> Hi there I was trying to run the project on my machine locally using maven, but when I do "mvn spring-boot:run" it gives error:
> 
> [ERROR] No plugin found for prefix 'spring-boot' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/Users/nirbhay/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
> 
> Thanks

You do not need Spring Boot to run the program, try running the command `& mvn javafx:run -f "path-to\Matchup-League\matchup\pom.xml"`

**Reply by NirbhaySharma504** [reply]
Hi
First of all thanks for answering!!
When i tried to run the command ``` mvn javafx:run -f "./Matchup-League-main/pom.xml" ```
I got this error:

[ERROR] Failed to execute goal org.openjfx:javafx-maven-plugin:0.0.8:run (default-cli) on project matchup: Error: Error: Output directory is empty -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.

I have tried to compile the project first too

**Reply by NirbhaySharma504** [reply]
> Hi First of all thanks for answering!! When i tried to run the command `mvn javafx:run -f "./Matchup-League-main/pom.xml"` I got this error:
> 
> [ERROR] Failed to execute goal org.openjfx:javafx-maven-plugin:0.0.8:run (default-cli) on project matchup: Error: Error: Output directory is empty -> [Help 1] [ERROR] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> 
> I have tried to compile the project first too

Okay so I tried copying the pom.xml into the path: ```./Matchup-League/matchup/ ``` and i did javafx run, it is working
But I am only able to access the menu, when i select a particular tournament to start, the game doesnt respond to my choice

**Reply by lognog2** [reply]
If it's getting stuck on the loading screen where it says "Connecting to database" it's likely a problem with Hibernate or mySQL. Otherwise check the console output or debug log to see what's happening

**Comment by freakun0025** [comment]
Hi thanks a lot for replying regarding the solution you gave, now for the error "output directory is empty is being shown" , reason for this might be because  there are no JavaFX-specific classes are present in your src/main/java directory. Correct me if I am wrong

**Reply by lognog2** [reply]
The JavaFX classes are located in java/com/menu
